src/main/java/com/oltpbenchmark/benchmarks/templated/TemplatedWorker.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 by OLTPBenchmark Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.oltpbenchmark.benchmarks.templated;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Map;
+
+import com.oltpbenchmark.api.Procedure;
+import com.oltpbenchmark.api.Procedure.UserAbortException;
+import com.oltpbenchmark.benchmarks.templated.procedures.GenericQuery;
+import com.oltpbenchmark.benchmarks.templated.util.TraceTransactionGenerator;
+import com.oltpbenchmark.api.TransactionType;
+import com.oltpbenchmark.api.Worker;
+import com.oltpbenchmark.types.TransactionStatus;
+
+public class TemplatedWorker extends Worker<TemplatedBenchmark> {
+
+    protected final Map<Class<? extends Procedure>, TraceTransactionGenerator> generators;
+
+    public TemplatedWorker(TemplatedBenchmark benchmarkModule, int id,
+                     Map<Class<? extends Procedure>, TraceTransactionGenerator> generators) {
+        super(benchmarkModule, id);
+        this.rng().setSeed(15721);
```

Comment:
We should probably pull this from the `workConf` instead of statically assigning it.

Reply:
The seed? I think I copied this from the TPC-H worker which has the same setup.

Reply:
Hmm, I wonder if that one is part of the TPC-H spec?
I think we should let it be config controllable here though.